spec.bs

```diff
@@ -846,6 +870,25 @@ input {{SecurePaymentConfirmationRequest}} |data|, are:
     matches, to defeat attempts to [[#sctn-privacy-probing-credential-ids|probe
     for credential existence]].
 
+1. Optionally, the user agent may remove entries from
+    |data|["{{SecurePaymentConfirmationRequest/facilitatingEntityLogos}}"],
+    starting from the back of the list and moving backwards.
+
+    Note: This allows the user agent to control how many logos it will display,
+    while preserving the semantics that logos are in decreasing order of
+    importance to the caller.
```

Comment:
Align "importance" here with whatever ends up being said above. For example: "in decreasing order of display priority to the caller."

Reply:
Done.